Chandra/cmd_states/cmd_states.py

```diff
@@ -382,9 +382,14 @@ def get_state0(date=None, db=None, date_margin=10, datepar='datestop'):
      time.  The conservative default value of ``date_margin=10`` allows for
      processing downtime.  The table is accessed via the ``db`` object.
 
+     If ``date_margin`` is None then no date margin is applied and the most
+     recent available state that meets the date selection criteria (including
+     that pcad_mode == 'NPNT') will be used.  In this case a ``date`` must be
```

Comment:
When you say "if statement", I assume now that you actually mean "if block"?

No, the above change does not work because of the `if date is None or date > definitive_date` test above, where `date` might not be an ISO-formatted date string at that point.